app/src/main/java/com/sdp/movemeet/models/Image.java

```diff
@@ -0,0 +1,71 @@
+package com.sdp.movemeet.models;
```

Comment:
Nice, we have a class for the image, it would be easier to use it now !

Reply:
Yes that's it! A new model class to work more easily with the Firebase Storage abstraction!